crates/dump/src/reader/snapshots/dump__reader__test__import_dump_v6_with_vectors-5.snap

```diff
@@ -49,6 +49,7 @@ expression: vector_index.settings().unwrap()
       "source": "huggingFace",
       "model": "BAAI/bge-base-en-v1.5",
       "revision": "617ca489d9e86b49b8167676d8220688b99db36e",
+      "pooling": "forceMean",
```

Comment:
Imported dump has its pooling set to `forceMean` for backward compatibility reasons